opcodes/riscv-dis.c

```diff
@@ -429,6 +429,7 @@ print_insn_args (const char *oparg, insn_t l, bfd_vma pc, disassemble_info *info
 	    print (info->stream, dis_style_immediate, "0");
 	  break;
 
+	case 'g':
```

Comment:
Missing break?

Reply:
This case is used by swap16 insn (`{"swap16",      0, INSN_CLASS_ZPN, "d,g",       MATCH_PKBT16, MASK_PKBT16, match_opcode, INSN_ALIAS }`). However I find it will be directly transformed into `PKBT16` by gas, such that objdump will never use this opcode. I'm going to delete this case.